src/server/minecraft-server.js

```diff
@@ -0,0 +1,1503 @@
+                            cachedPlayer.opped = await this.determineOpStatus(cachedPlayer);
+                            cachedPlayer.whitelisted = await this.determineWhitelistStatus(cachedPlayer);
+                            cachedPlayer.online = false;
+                            playersList.players.push(cachedPlayer);
+                        }
+                    }
+                } else {
+                    cachedPlayer.key = cachedPlayer.uuid;
+                    cachedPlayer.banned = await this.determineBanStatus(cachedPlayer);
+                    cachedPlayer.opped = await this.determineOpStatus(cachedPlayer);
+                    cachedPlayer.whitelisted = await this.determineWhitelistStatus(cachedPlayer);
+                    cachedPlayer.online = false;
+                    playersList.players.push(cachedPlayer);
+                }
+            }
+            properties.playerInfo = playersList;
+
+            return Promise.resolve(playersList);
+        } else {
+            return Promise.resolve(playersList);
+        }
+    }
+
+    async listWorldBackups () {
+        await this.log('Getting list of MinecraftServer world backups...');
+
+        let properties = this.properties;
+        let settings = properties.settings;
+        let backupDir = path.resolve(settings.backups.path);
+        let backupList = [];
+        let files = [];
+
+        return new Promise(async (resolve, reject) => {
+            properties.backupList = [];
+            try {
+                files = await fs.readdir(backupDir)
+            } catch (err) {
+                if (err.code === 'ENOENT') {
+                    resolve(properties.backupList)
+                }
+                await this.log(`There was a problem accessing ${backupDir}.`);
+                reject(err);
+            }
+            files.forEach(file => {
+                let fileInfo,
+                    fileItem = {},
+                    fileParts = file.split('.');
+    
+                if (fileParts[1] === 'zip') {
+                    fileInfo = fileParts[0].split('_');
+                    fileItem.fileName = file;
+                    fileItem.worldName = fileInfo[0];
+                    fileItem.date = fileInfo[1];
+                    fileItem.time = fileInfo[2];
+                    backupList.push(fileItem);
+                }
+            });
+            properties.backupList = backupList;
+            resolve(properties.backupList);
+        });
+    }
+
+    async deleteWorldBackups () {
+        let properties = this.properties;
+        let settings = properties.settings;
+        let backupDir = path.resolve(settings.backups.path);
+
+        return new Promise(async (resolve, reject) => {
+            await this.listWorldBackups();
+            for (let backup of properties.backupList) {
+                try {
+                    await fs.unlink(path.resolve(path.join(backupDir, backup.fileName)));
```

Comment:
`fs-extra.remove` would avoid unnecessary 'File not found' errors. I won't mark the remaining `unlink` calls, you can replace them as you see fit